pom.xml

```diff
@@ -51,7 +51,7 @@
         <esh.version>0.8.0.b2</esh.version>
         <shk.version>1.2</shk.version>
         <ohc.version>2.0.0.b1</ohc.version>
-        <tp.version>2.0.0.a1</tp.version>
+        <tp.version>2.0.0.b2</tp.version>
```

Comment:
Works for me. That's actually how I started, but then for some reason, I changed it...